### DISCUSSION TITLE
Internal server error 500, new install

### DISCUSSION THREAD

**Discussion by Kyian2000** [firefly-iii/firefly-iii#9074] [Firefly III]
Hi all, 

I just installed the FireFly, and when I look in container, it says FireFly_iii_core: unhealthy, when I go to the website I get this error. Not sure, what's really wrong, as I followed the latest guide.

I don't receive any error code, when building it. 

```
Firefly III - 500 Internal Server Error :(
Whoops! An error occurred.
Unfortunately, this error was not recoverable :(. Firefly III broke. The error is:

Could not poll the database: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'firefly.configuration' doesn't exist (Connection: mysql, SQL: select `id`, `name`, `data` from `configuration` where `name` = is_demo_site and `configuration`.`deleted_at` is null limit 1)

This error occured in file /var/www/html/app/Support/FireflyConfig.php on line 65 with code 0.
```

**Comment by JC5** [comment]
It looks like you didn't run the migrations. Did you follow all the installation steps?

**Comment by Kyian2000** [comment]
Yep, followed all the steps on this one: https://docs.firefly-iii.org/how-to/data-importer/installation/docker/

I'm running it on a Synology NAS. In the guide, it doesn't say anything about migration?

**Reply by JC5** [reply]
Mmm no, you wouldn't have to since Docker should do this automatically. What happens when you (re)start the container? Could you share the full logs?

**Reply by Kyian2000** [reply]
I just tried it with portainer as well, same error comes. Heres the full lof after reboot. 

[firefly_iii_core.csv](https://github.com/user-attachments/files/16331955/firefly_iii_core.csv)

**Reply by JC5** [reply]
Mmm, I see that it runs the migrations just fine. But one of the containers stops because of SIGWINCH? I can't be sure why it stops.

**Reply by Kyian2000** [reply]
Looks like it got it self sorted, now it's just the localhost that doesn't work and the data importer. When I try to add the Client ID

![Screenshot 2024-07-22 at 13 25 55](https://github.com/user-attachments/assets/13be7e2d-3655-4a90-931b-d00370d81e75)

**Reply by JC5** [reply]
Make sure you set the FIREFLY_III_URL and VANITY_URL to the correct value.

**Reply by Kyian2000** [reply]
I used what the guide said, with localhost. And App

**Reply by Kyian2000** [reply]
So i changed it to this, but that might be wrong. 

Change FIREFLY_III_URL in .importer.env to http://app:8080
Change VANITY_URL in .importer.env to http://localhost

**Reply by Kyian2000** [reply]
Any suggestions?

**Reply by JC5** [reply]
Which address is Firefly III running from?

**Comment by Kyian2000** [comment]
It’s running from 192.168.12.100, tried changing the addresses to that as well. Then on port 901 and 902. Den 25. jul. 2024 kl. 08.17 skrev James Cole ***@***.***>:﻿
Which address is Firefly III running from?

**Reply by JC5** [reply]
Mmmm, that should do it. `FIREFLY_III_URL=http://app:8080`, and `VANITY_URL=http://192.168.12:100:port`. Be sure to press the "start over" button.
